src/background/tab-manager.ts

```diff
@@ -79,6 +79,23 @@ export default class TabManager {
                 a.download = name;
                 a.click();
             }
+            if (type === 'export-css-proxy') {
+                const tabs = await queryTabs({});
+                const activeURL = await this.getActiveTabURL();
```

Comment:
Have a look at `getActiveTabURL` function. Rename it into `getActiveTab` and remove `.url` from return value. Add `getActiveTabURL` function that will look like `const tab = await getActiveTab()); return tab.url`.

Use `getActiveTab` in this case.